src/components/Camera/style.css

```diff
@@ -103,6 +102,21 @@
   /*Mirroring transform: scale(-1, 1); */
 }
 
+@supports (object-fit: cover){
+  .video{height: 100%; width:100%}
+}
+/* this will only apply styles to Edge (and IE)*/
+@supports (object-fit: cover) and (-ms-ime-align:auto) {
```

Comment:
should it not be `supports not`? I'm a bit confused

Reply:
Good spot. I had both queries because I had some properties that were shared between Edge and other browsers but I removed them. I just need `@supports (-ms-ime-align:auto) {...}`

Reply:
Actually, you are right, it should be `not`.

Reply:
The reason for having both properties is that Edge has `object-fit: cover` support for `<img>` not for `<video>` while IE11 doesn't have `object-fit` support at all. So this will target Edge only. I will change the comment and the PR description